TestingTool/scenario.py

```diff
@@ -4,7 +4,26 @@
 from mongoengine import StringField, EmbeddedDocumentListField
 from .task import Task
 from .result import Result
+from selenium import webdriver
 
 
 class Scenario(EmbeddedDocument):
-    pass  # I'm just a placeholder
+    name = StringField(max_length=128, required=True)
+    url = StringField(max_length=512, required=True)
+    tasks = EmbeddedDocumentListField(Task)
+
+    def run(self):
+        driver = webdriver.Firefox()
+        driver.implicity_wait(5)
+        driver.maximize_window()
+        data = []
+        result = True
+        for task in self.tasks:
+            task.run(driver)
+            result = task.result.result
+            if result is False:
+                data.append(task.result.data)
+        if data:
+            result = False
+        response_result = Result(result, data)
```

Comment:
Result(result, data) este ambiguu, ar trebui specificat carui atribut ar trebui sa ii fie asignat si ce anume, spre exemplu
`Result(result=result, data=data)` unde ce e la stanga egalului e atributul asa cum este denumit in clasa Result, si ce e in dreapta este obiectul asignat lui.